include/nrsc5.h

```diff
@@ -320,6 +336,7 @@ struct nrsc5_event_t
             const char *artist;
             const char *album;
             const char *genre;
+            nrsc5_id3_comm *comments;
```

Comment:
This should probably go at the end of the struct. Inserting it in the middle will break ABI compatibility.